### PR TITLE
Bumped jackson version

### PR DESCRIPTION
**Pull request by jebeaudet**

### CODE REVIEW

File: base64 [hunk 1]
```diff
@@ -0,0 +1 @@
+JSAdmin:FqY%uDQPPP7s4fbxs6qV9GeFPLFdfLDkCjMW;banuC+h>mtYRC --encode
```
Comment (fredboutin) [inline base64:1]: what is this?

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.coveo</groupId>
     <artifactId>spring-boot-parameter-store-integration</artifactId>
-    <version>1.3.1</version>
+    <version>1.4.0-SNAPSHOT</version>
```
Comment (fredboutin) [inline pom.xml:8]: 1.4.0 and don't forget the readme

File: pom.xml.versionsBackup [hunk 3]
```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fredboutin) [inline pom.xml.versionsBackup:1]: delete this

File: pom.xml (hunk 2 above)
Comment (fredboutin) [reply pom.xml:8]: i can do the release if you want, if you do it, don't forget to do the release in github also

Comment (jebeaudet) [reply pom.xml:8]: It's just for snyk, I'll leave snapshot for now for some time

**Pull request merged**